src/styles/blocks/card.scss

```diff
@@ -0,0 +1,123 @@
+.cards {
+  display: grid;
+  grid-gap: 46px 48px;
+  padding: 50px $space-40;
+}
+
+.card {
+  margin-bottom: 0;
+  width: 200px;
+
+  background-color: $button-light-color;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+}
+
+.card > img {
+  width: 160px;
+  display: block;
+  margin: 32px auto 0;
+}
+
+.card__details {
+  margin: $space-40 16px 0;
+}
+
+.card__details-title {
+  font-size: $font-size-12;
+  font-weight: $font-weight-500;
+  line-height: $line-height-18;
+  color: $main-color;
+  margin: 0;
+
+  &:hover {
+    color: $title-hover-color;
+    transition-duration: $duration-time;
+  }
+}
+
+.card__details-code {
+  margin: 0;
+  padding-top: 5px;
```

Comment:
I think it would be better to use only margins for setting spacing between elements